### PR TITLE
Practice

### PR DESCRIPTION
**Pull request by hello-dino**
# Motivation

- Please explain why you're making this change and what problem you're trying to solve.

# Modifications

- Please list the modifications you've made in detail.

# Commit Convention Rule

- Please commit your modification based by [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/):

| Commit type | Description                                                                     |
|-------------|---------------------------------------------------------------------------------|
| feat        | New Feature                                                                     |
| fix         | Fix bug                                                                         |
| docs        | Documentation only changed                                                      |
| ci          | Change CI configuration                                                         |
| refactor    | Not a bug fix or add feature, just refactoring code                             |
| test        | Add Test case or fix wrong test case                                            |
| style       | Only change the code style(ex. white-space, formatting)                         |
| chore       | It refers to minor tasks such as library version upgrade, typo correction, etc. |

- If you want to add some more `commit type` please describe it on the **Pull Request**

# Result

- Please describe the result after this PR is merged.

# Closes

- #{issue number} (If this PR resolves an issue.)

### CODE REVIEW

File: dino1-1
Comment (hello-dino) [inline dino1-1:1]: HELLO

Comment (hello-dino) [reply dino1-1:1]: WORLD

Comment (hello-dino) [reply dino1-1:1]: HOHO

**Review by hello-dino — COMMENTED** [review]
LGTM

**Comment by waahhh** [comment]
LGTM

**Pull request merged**